diff --git a/node_modules/@matterlabs/zksync-nuxt-core/types/index.ts b/node_modules/@matterlabs/zksync-nuxt-core/types/index.ts
index 2eb2e52..322959f 100644
--- a/node_modules/@matterlabs/zksync-nuxt-core/types/index.ts
+++ b/node_modules/@matterlabs/zksync-nuxt-core/types/index.ts
@@ -18,7 +18,7 @@ import { Initialization as OnboardOptions } from "bnc-onboard/dist/src/interface
 
 export type Link = `https://${string}/`;
 
-export type EthereumNetwork = "rinkeby" | "ropsten" | "mainnet";
+export type EthereumNetwork = "rinkeby" | "ropsten" | "mainnet" | "localhost";
 
 export type ZkEthereumNetworkName = "localhost" | "rinkeby" | "rinkeby-beta" | "ropsten" | "ropsten-beta" | "mainnet";
 
diff --git a/node_modules/@matterlabs/zksync-nuxt-core/utils/config.ts b/node_modules/@matterlabs/zksync-nuxt-core/utils/config.ts
index 708ae84..e1a0180 100644
--- a/node_modules/@matterlabs/zksync-nuxt-core/utils/config.ts
+++ b/node_modules/@matterlabs/zksync-nuxt-core/utils/config.ts
@@ -6,8 +6,8 @@ import { onboardConfig } from "../utils/onboardConfig";
 
 export const zkSyncNetworkConfig: ZkNetworkConfig = {
   localhost: {
-    ethereumNetwork: "rinkeby",
-    api: "https://rinkeby-api.zksync.io/api/v0.2/",
+    ethereumNetwork: "localhost",
+    api: "https://127.0.0.1:3001/api/v0.2/",
     explorer: "https://rinkeby.zkscan.io/",
     tools: {
       forcedExit: "https://withdraw-rinkeby.zksync.dev/",
@@ -93,6 +93,12 @@ export const ethereumNetworkConfig = (INFURA_KEY: string): ZkEthereumNetworkConf
       explorer: "https://etherscan.io/",
       rpc_url: `https://mainnet.infura.io/v3/${INFURA_KEY}`,
     },
+    localhost:{
+      id: 33,
+      name: "localhost",
+      explorer: "https://rinkeby.etherscan.io/",
+      rpc_url: `https://rinkeby.infura.io/v3/${INFURA_KEY}`,
+    }
   };
 };
 
